mautic/mautic
# Pipedrive full sync

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5164
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

This bugfix/feature has been sponsored by @Webmecanik

[//]: # ( Required: )
#### Description:
PR added missed sync between Mautic and Pipdive
From this request: https://github.com/mautic/mautic/issues/5164

#### Steps to test this PR:
1.  Setup https://mautic.org/docs/en/plugins/pipedrive.html (for test trial is useful)
2.  Create contact in Pipedrive and see if contact was created in Mauitc
3.  Edit contact in Pipedrive and see if contact was modified in Mautic
4. Create contact in Mautic and see if contact was created  in Pipedrive
5. Modify contact in Mautic and see if contact was modifeid in Pipedrive
6.  Disable Pipedrive plugin
7. Create new contact in Mautic and another contact in Pipdedrive. 
8. Enable Pipedrive plugin
9. Go to Mautic and edit contact created while plugin was disabled (step 6 ). Contact should  be created in Pipedrive
10. Go to Pipedrive and edit contact create while plugin was disabled (step 6).  Contact should be created in Mautic
11.  Disable Pipedrive plugin
12. Go to Mautic and Pipedrive and create new contact (same for both systems)
13. Enable Pipedrive plugin
14. Try edit contact in Mautic, should by update also on Pipedrive
